lib/Doctrine/DBAL/Platforms/MySqlPlatform.php

```diff
@@ -291,7 +269,7 @@ public function getClobTypeDeclarationSQL(array $field)
      */
     public function getDateTimeTypeDeclarationSQL(array $fieldDeclaration)
     {
-        if (isset($fieldDeclaration['version']) && $fieldDeclaration['version'] == true) {
+        if (isset($fieldDeclaration['version']) && $fieldDeclaration['version'] === true) {
```

Comment:
This can be considered a BC break. Could be replaced with `if (!empty($fieldDeclaration['version']))`.